src/components/Composer/index.native.js

```diff
@@ -90,6 +94,17 @@ class Composer extends React.Component {
         this.props.onClear();
     }
 
+    onChangeText(text) {
+        this.setState({value: text});
+        this.props.onChangeText(text);
+    }
+
+    setTextAndSelection(text, start, end) {
+        this.setState({value: text}, () => {
+            this.textInput.setSelection(start, end);
```

Comment:
is it safe to use this internal method?

Reply:
I'd say yes as it's publicly available on the `TextInput`'s ref. In the future, the react-native team might migrate that to `setSelectionRange` to match the web's implementation name (as hinted here: https://github.com/react-native-community/discussions-and-proposals/pull/496)

Reply:
Ok. Thanks.